presidential-elections/utils/model_builder.py

```diff
@@ -517,14 +520,28 @@ def _build_data_containers(
         self,
         polls: pd.DataFrame = None,
         campaign_predictors: pd.DataFrame = None,
-    ) -> Dict[str, pm.Data]:
+    ) -> Tuple[Dict[str, pm.Data], Dict[str, np.ndarray]]:
 
         if polls is None:
             polls = self.polls_train
         if campaign_predictors is None:
             campaign_predictors = self.campaign_preds
 
-        return dict(
+        is_here = polls[self.parties_complete].astype(bool).astype(int)
+        non_competing_parties = {
+            "polls_multiplicative": is_here.values,
+            "polls_additive": is_here.replace(to_replace=0, value=-10)
+                .replace(to_replace=1, value=0)
+                .values,
+            "results": self.results_mult[self.parties_complete]
+                .astype(bool)
+                .astype(int)
+                .replace(to_replace=0, value=-10)
```

Comment:
What does `-10` stand for here? :thinking:

Reply:
It's just an arbitrary placeholder to make sure the probability for the non-competing parties will be very close to 0 once we take the softmax (-10 is very low compared to the other partie's values, so that does the trick)